docs/guides/nginx-startup.md

```diff
@@ -255,21 +255,21 @@ An additional Redis service is not intended for production environments where a
 For that reason also the PWA Helm chart does not support deploying an own Redis service with the PWA.
 Only the `redis.uri` can be configured for a Helm deployment.
 
-#### Brotli configuration
+### Brotli configuration
 
 All [configuration directives](https://github.com/google/ngx_brotli?tab=readme-ov-file#configuration-directives) can be set in
-[compression.conf](../../nginx/features/compression.conf).
+[`compression.conf`](../../nginx/features/compression.conf).
 
-The on-the-fly compression Brotli compression level is set to the [recommended](https://www.brotli.pro/enable-brotli/servers/nginx/) value of `4` instead of 6 ([default](https://github.com/google/ngx_brotli?tab=readme-ov-file#brotli_comp_level)).
+The on-the-fly compression Brotli compression level is set to the [recommended](https://www.brotli.pro/enable-brotli/servers/nginx/) value of `4` instead of `6` ([default](https://github.com/google/ngx_brotli?tab=readme-ov-file#brotli_comp_level)).
 This setting provides a balance between the compression rate and CPU usage.
 A higher level would achieve better compression but would also consume more CPU resources.
 
 The two NGINX modules
 
-- ngx_http_brotli_filter_module.so – for compressing responses on-the-fly
-- ngx_http_brotli_static_module.so - for serving pre-compressed files
+- `ngx_http_brotli_filter_module.so` – for compressing responses on-the-fly
+- `ngx_http_brotli_static_module.so` - for serving pre-compressed files
 
-are built in the [Dockerfile](../../nginx/Dockerfile) using an NGINX archive in a version which matches the openresty Docker image version and referenced in [nginx.conf](../../nginx/nginx.conf).
+are built in the [`Dockerfile`](../../nginx/Dockerfile) using an NGINX archive in a version which matches the openresty Docker image version and referenced in [`nginx.conf`](../../nginx/nginx.conf).
```

Comment:
NGINX is lowercase elsewhere in this document (e.g., "nginx container"). Should it always be capitalized?